core-collapse.html

```diff
@@ -239,10 +247,11 @@
         return;
       }
       if (!this.isTargetReady) {
-        this.targetChanged(); 
+        this.targetChanged();
       }
       this.horizontalChanged();
       this[this.opened ? 'show' : 'hide']();
+      this.notifyResize();
```

Comment:
Do you need to call `notifyResize` here?  There is already a call to `notifyResize` in `transitionEnd` and it seems that should be enough.